samples/meeting-av-filters/js/AppPackage/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
The manifest has been updated. see https://github.com/microsoft/teams-videoapp-sample/blob/main/manifest/manifest.json
changes:
- $schema
- remove "devicePermission"

Reply:
Updated the manifest